### PR TITLE
Add a Dockerfile for elasticsearch 6

### PR DESCRIPTION
**Pull request by gpocentek**
It is based on the official elastic images:
https://www.elastic.co/guide/en/elasticsearch/reference/current/docker.html

### CODE REVIEW

File: docker/images/elasticsearch/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,3 @@
+FROM docker.elastic.co/elasticsearch/elasticsearch:6.2.4
+
+USER 1001
```
Comment (jmaupetit) [inline docker/images/elasticsearch/Dockerfile]: Why this user ID?

Comment (jmaupetit) [inline docker/images/elasticsearch/Dockerfile]: Why is this required if OpenShift already changes the default user when running this image?

**Comment by jmaupetit** [comment]
Also, don't forget to update the documentation (README) for this new version of service.

File: docker/images/elasticsearch/Dockerfile (hunk 1 above)
Comment (gpocentek) [reply docker/images/elasticsearch/Dockerfile]: This helps openshift detect that the container isn't running as root. Not sure entirely why this is needed but this make the image usable in openshift.

1001 is just a random choice, could be anything else.

Comment (jmaupetit) [reply docker/images/elasticsearch/Dockerfile]: Thank you for this answer! Can you add this as a comment then?

**Review by jmaupetit — APPROVED** [review]
Thx!

**Review by sampaccoud — APPROVED** [review]

**Comment by jmaupetit** [comment]
Hey @gpocentek looks like our CI is pending. I know that CircleCI encountered an outage recently. Can you rebase this branch just to change commit ids and force push your branch?
